neurodamus/node.py

```diff
@@ -1151,7 +1151,6 @@ def _sim_init_neuron(self):
             with timeit(name="restoretime"):
                 logging.info("Restoring state...")
                 self._stim_manager.saveStatePreparation(self._bbss)
-                self._stim_manager.reevent()
```

Comment:
This reevent() is not needed anymore?

Reply:
yes, in StimulusManager.hoc, reevent() is to execute `synapseEventGenerator` which is set by  `NPoissonInhomogeneous` stimulus. In SONATA, we don't have such stimulus any more.